src/core/Microsoft.PowerToys.Settings.UI.Lib/SettingsUtils.cs

```diff
@@ -14,16 +14,23 @@ public static class SettingsUtils
         /// <returns>string path.</returns>
         public static string GetSettingsPath(string powertoy)
         {
+            string dir;
             if (string.IsNullOrWhiteSpace(powertoy))
             {
-                return Path.Combine(
-                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
-                    $"Microsoft\\PowerToys\\settings.json");
+                dir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), $"Microsoft\\PowerToys\\");
+            }
+            else
+            {
+                dir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), $"Microsoft\\PowerToys\\{powertoy}\\");
+            }
+
+            // create dirctory if one does not exist.
+            if (!Directory.Exists(dir))
```

Comment:
Suggestion as a single intended PR can we separate fixing an issue when directory not exist in a separate PR.

Reply:
the icons are only temporary. It was to avoid having pages with the same icon on the side.